frontend/src/app/dataquery.ts

```diff
@@ -40,7 +43,9 @@ export abstract class DataQuery {
   constructor() {
     this.http = InjectorInstance.get<HttpClient>(HttpClient);
   }
+}
 
+export abstract class DataQuery extends Query {
   public abstract loadTablePage(
     offset: number,
     limit: number
```

Comment:
Das sieht in der Differenz recht verwirrend aus. Vorher war DataQuery eine gemeinsame Elternklasse für ganz viele Abfragen die wir gemacht hat, die brauchte aber eine Implementierung für loadTablePage und loadRowCount was in einigen Fällen kein Sinn gemacht hat. Deshalb gibt es jetzt eine Elternklasse Query, die die wichtigsten Dinge aus DataQuery auslagert.

Reply:
Die Query benutzen wir ansonsten nur, wenn wir Datenbankenspalten haben wollen (etwa für die Table Preview oder für ViolatingFDs), würde die hier nicht verwenden. Und wenn du von Query statt DataQuery erbst nenn die Klasse bitte nicht DataQuery^^. Würde aber direkt die http-Befehle in die `schema-creation.service.ts` integrieren